src/main/java/com/alkemy/ong/controller/SlideController.java

```diff
@@ -0,0 +1,43 @@
+package com.alkemy.ong.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.alkemy.ong.model.Slide;
+import com.alkemy.ong.service.SlideService;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@RestController
+@RequestMapping("/slides")
+public class SlideController {
+
+    @Autowired
+    SlideService slideService;
+
+    @GetMapping
+    public ResponseEntity<List<SlideDto>> getAllCategories() {
+        List<Slide> slides = slideService.getAllSlides();
+        List<SlideDto> slideDtos = slides.stream().map(
```

Comment:
Todo esto lo deberias hacer del servicio, y ya devolver la lista de slideDto, esta bien que hayas usado el map con el lambda. Lo que is subime el slideDto porque no esta en main y si lo mergeo se va a romper todo